lib/CGameState.cpp

```diff
@@ -2090,6 +2090,7 @@ void CGameState::updateRumor()
 			}
 			else
 				rumor.type = RumorState::TYPE_RAND;
+			//-fallthrough
```

Comment:
Is there any documentation for such syntax?

Reply:
https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html
-Wimplicit-fallthrough=3 case sensitively matches one of the following regular expressions:
    -fallthrough